src/main.c

```diff
@@ -526,7 +527,12 @@ init_config(struct cfg *cf, int argc, char **argv)
            printf("%s\n", get_mclock_name());
            rtpp_exit(0);
            break;
-
+  case 'B':
```

Comment:
We need to make sure this is in the service file as "-B" too. If it's already there, it may be "-b".